CHANGELOG.md

```diff
@@ -4,6 +4,82 @@
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
 
+## [0.10.0](https://github.com/rerun-io/rerun/compare/0.9.1...0.10.0) - C++ SDK - 2023-10-30
+
+[Rerun](https://www.rerun.io/) is an easy-to-use visualization toolbox for computer vision and robotics.
+
+* Python: `pip install rerun-sdk`
+* Rust: `cargo add rerun` and `cargo install rerun-cli`
+* Online demo: <https://app.rerun.io/version/0.9.1/>
+
+### Overview & Highlights
+* The C++ SDK is finally here!
+* Add an integrated getting-started guide into the viewer splash screen
+* Add a new and improved `spawn` method in the C++ SDK
```

Comment:
from the pov of a newcomer that's just part of the **initial** C++ bundle. But that it's new on Rust is worth calling out